base_tier_validation/models/tier_validation.py

```diff
@@ -221,7 +221,9 @@ def _compute_next_review(self):
             review = rec.review_ids.sorted("sequence").filtered(
                 lambda x: x.status == "pending"
             )[:1]
-            rec.next_review = review and _("Next: %s") % review.name or ""
+            rec.next_review = review and self.env._(
+                "Next: %(review_name)s", review_name=review.name or ""
+            )
```

Comment:
Same as previous comments. It's a shame to invalidate existing translatable terms just to use a kword argument on a term with only 1 arg